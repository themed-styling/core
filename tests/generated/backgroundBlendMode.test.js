import 'jest-styled-components'

import { backgroundBlendMode } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(backgroundBlendMode, 'backgroundBlendMode')
