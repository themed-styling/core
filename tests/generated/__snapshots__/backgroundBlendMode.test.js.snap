// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`backgroundBlendMode renders 0 1`] = `
Array [
  .c0 {
  background-blend-mode: 0;
}

<div
    className="c0"
  />,
  .c0 {
  background-blend-mode: 0;
}

<div
    className="c0"
  />,
  .c0 {
  background-blend-mode: 0;
}

<div
    className="c0"
  />,
  .c0 {
  background-blend-mode: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`backgroundBlendMode renders function 1`] = `
Array [
  .c0 {
  background-blend-mode: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  background-blend-mode: has;
}

<div
    className="c0"
  />,
  .c0 {
  background-blend-mode: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  background-blend-mode: has;
}

<div
    className="c0"
  />,
]
`;

exports[`backgroundBlendMode renders invalid theme path 1`] = `
Array [
  .c0 {
  background-blend-mode: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  background-blend-mode: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  background-blend-mode: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  background-blend-mode: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`backgroundBlendMode renders negative number 1`] = `
Array [
  .c0 {
  background-blend-mode: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  background-blend-mode: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  background-blend-mode: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  background-blend-mode: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`backgroundBlendMode renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    background-blend-mode: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    background-blend-mode: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    background-blend-mode: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    background-blend-mode: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    background-blend-mode: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    background-blend-mode: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    background-blend-mode: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    background-blend-mode: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    background-blend-mode: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    background-blend-mode: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    background-blend-mode: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    background-blend-mode: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    background-blend-mode: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    background-blend-mode: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    background-blend-mode: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    background-blend-mode: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`backgroundBlendMode renders object [breakpoints] 2`] = `
Array [
  @media May {
  .c0 {
    background-blend-mode: July;
  }
}

@media April {
  .c0 {
    background-blend-mode: August;
  }
}

@media January {
  .c0 {
    background-blend-mode: September;
  }
}

@media April {
  .c0 {
    background-blend-mode: November;
  }
}

@media June {
  .c0 {
    background-blend-mode: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    background-blend-mode: July;
  }
}

@media April {
  .c0 {
    background-blend-mode: August;
  }
}

@media January {
  .c0 {
    background-blend-mode: September;
  }
}

@media April {
  .c0 {
    background-blend-mode: November;
  }
}

@media June {
  .c0 {
    background-blend-mode: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    background-blend-mode: January;
  }
}

@media April {
  .c0 {
    background-blend-mode: December;
  }
}

@media January {
  .c0 {
    background-blend-mode: April;
  }
}

@media April {
  .c0 {
    background-blend-mode: May;
  }
}

@media June {
  .c0 {
    background-blend-mode: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    background-blend-mode: January;
  }
}

@media April {
  .c0 {
    background-blend-mode: December;
  }
}

@media January {
  .c0 {
    background-blend-mode: April;
  }
}

@media April {
  .c0 {
    background-blend-mode: May;
  }
}

@media June {
  .c0 {
    background-blend-mode: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`backgroundBlendMode renders postitive number 1`] = `
Array [
  .c0 {
  background-blend-mode: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  background-blend-mode: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  background-blend-mode: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  background-blend-mode: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`backgroundBlendMode renders string 1`] = `
Array [
  .c0 {
  background-blend-mode: July;
}

<div
    className="c0"
  />,
  .c0 {
  background-blend-mode: July;
}

<div
    className="c0"
  />,
  .c0 {
  background-blend-mode: June;
}

<div
    className="c0"
  />,
  .c0 {
  background-blend-mode: June;
}

<div
    className="c0"
  />,
]
`;

exports[`backgroundBlendMode renders valid theme path 1`] = `
Array [
  .c0 {
  background-blend-mode: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  background-blend-mode: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  background-blend-mode: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  background-blend-mode: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
